.github/workflows/test-build-spack.yml

```diff
@@ -80,9 +80,10 @@ jobs:
     - name: Install Core Development Tools
       run: |
         sudo apt-get -qq update
+        sudo add-apt-repository ppa:ubuntu-toolchain-r/test
```

Comment:
This is an untrusted PPA (Personal Package Archive). This introduces a security risk if any sensitive information (keys, passwords, etc) are ever entered into the GitHub runner ... including the ssh keys used by git clone and push actions. Can you find an alternative or skip using gfortran-13?